client/blocks/jitm/templates/sidebar-banner.jsx

```diff
@@ -3,39 +3,44 @@ import { preventWidows } from 'calypso/lib/formatting';
 
 import './sidebar-banner.scss';
 
-export default function SidebarBannerTemplate( {
-	CTA,
-	message,
-	id,
-	dismissPreferenceName,
-	onDismissClick,
-	tracks,
-	tracksDismissName,
-	tracksDismissProperties,
-} ) {
+export default function SidebarBannerTemplate( { CTA, message, id, onDismiss, tracks, ...props } ) {
```

Comment:
[In the past](https://github.com/Automattic/wp-calypso/pull/38881#discussion_r370402507), we've recommended to maintain explicit props rather than allowing arbitrary props. 

Seems we only check `props.isDismissible` below, so it may be better to note it here:
```suggestion
export default function SidebarBannerTemplate( { CTA, message, id, onDismiss, tracks, isDismissible = false } ) {
```